test/metabase/api/dashboard_test.clj

```diff
@@ -1268,6 +1268,12 @@
   [id]
   (t2/hydrate (t2/select-one DashboardCard :id id) :series))
 
+(defn- current-tabs
+  "Returns the current ordered tabs of a dashboard."
+  [dashboard-id]
+  (-> dashboard-id
+      dashboard/ordered-tabs))
```

Comment:
This seems unnecessary, just use `dashboard/ordered-tabs`

Reply:
updated in e57d014241